packages/@aws-cdk/aws-apigateway/test/test.method.ts

```diff
@@ -578,5 +578,32 @@ export = {
     }));
 
     test.done();
-  }
+  },
+
+  'use default requestParameters'(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const api = new apigateway.RestApi(stack, 'test-api', {
+      cloudWatchRole: false,
+      deploy: false,
+      defaultMethodOptions: {
+        requestParameters: {"method.request.path.proxy": true}
+      }
+    });
+
+    // WHEN
+    new apigateway.Method(stack, 'my-method', {
+      httpMethod: 'POST',
+      resource: api.root,
+    });
+
+    // THEN
+    expect(stack).to(haveResource('AWS::ApiGateway::Method', {
```

Comment:
Could you use another property from the `MethodProps` of 'my-method' to ensure you're validating the correct `AWS::ApiGateway::Method` in case another one is added later to this test case?
You could just use the `httpMethod`, but even better if you can set the `operationName` and verify that.

Reply:
Done :)